fsspec/fuse.py

```diff
@@ -164,3 +213,122 @@ def run(fs, path, mount_point, foreground=True, threads=False):
             func()
         except KeyboardInterrupt:
             pass
+
+
+def mount(
+    url,
+    source_path,
+    mount_point,
+    log_file,
+    option,
+    foreground=True,
+    threads=False,
+    ready_file=False,
+):
+    kwargs = {}
+    for item in option:
+        key, value = item.split("=", 1)
```

Comment:
Use `str.partition` for better error messages when the `item` doesn't include `=`.

Reply:
Good idea!